docs/topics/wasm/wasm-js-interop.md

```diff
@@ -451,8 +451,6 @@ generic `WebAssembly.Exception` without directly accessible messages and data.
 
 ## Kotlin/Wasm and Kotlin/JS interoperability differences
 
-<a name="differences"/>
-
```

Comment:
JFYI
the content which refer this link was deleted in commit ad3ebe2500886ebbe082dfd6b5dd7db2823144a0

so I think this link itself can be deleted to

Reply:
For the reviwer: double-check that there is no external link via kotl.in in IDEA/AS to the anchor